src/cli.ts

```diff
@@ -1,22 +1,52 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as glob from 'glob';
+import * as reflect from 'jsii-reflect';
 import * as yargs from 'yargs';
-import { renderFiles, renderSinglePageModule } from './index';
+import { Documentation } from './index';
 
 export async function main() {
   const args = yargs
-    .usage('Usage: $0 [JSII-FILE,...]')
-    .option('output', { type: 'string', alias: 'o', required: false, desc: 'Output directory or file (default directory is "dist", default file is "APP.md")' })
+    .usage('Usage: $0')
+    .option('output', { type: 'string', alias: 'o', required: false, desc: 'Output filename (defaults to API.md)' })
     .example('$0', 'Generate documentation for the current module as a single file (auto-resolves node depedencies)')
-    .example('$0 node_modules/**/.jsii', 'Generate documentation for all jsii modules in your project')
     .argv;
 
+  const assembly = await createAssembly();
 
-  if (args._.length === 0) {
-    const output = args.output ?? 'API.md';
-    await renderSinglePageModule(process.cwd(), output);
-  } else {
-    const output = args.output ?? 'dist';
-    await renderFiles(args._ as string[], output);
+  const docs = new Documentation({
```

Comment:
Since generating the API reference is slightly more involved now, requiring dependant assemblies, and operating on a single package, I removed support for passing any input to the CLI.

I think the main use-case for this tool is to generate the `API.md` file as part of the normal jsii project build procedure, so this is what it currently supports.